toolchain/check/eval_inst.cpp

```diff
@@ -516,6 +516,17 @@ auto EvalConstantInst(Context& context, SemIR::InstId inst_id,
   return ConstantEvalResult::NewSamePhase(inst);
 }
 
+auto EvalConstantInst(Context& /*context*/, SemIR::RequireDecl inst)
+    -> ConstantEvalResult {
+  return ConstantEvalResult::NewSamePhase(SemIR::RequireDecl{
+      .type_id = inst.type_id,
+      .require_impls_id = inst.require_impls_id,
+      // The constant value doesn't include the body block so that we don't try
+      // to import it, and it doesn't pollute the textual semir.
```

Comment:
```suggestion
      // to import it, and it doesn't pollute the textual SemIR.
```

nit: TBH I find the capitalization helpful, it means you aren't referring to the "semir" object in specific, but rather the more abstract SemIR. Either way, I try to capitalize proper nouns/acronyms in comments.

Reply:
Done